### PR TITLE
Add rocksdb configs

### PR DESCRIPTION
**Pull request by overvenus**
#601 

-----
The getopts in crate.io is outdated, use the latest getopts instead. In the old version getopts, It seems like there is no way to avoid panic if short is not defined.

### CODE REVIEW

File: etc/config-template.toml [hunk 1]
```diff
@@ -27,4 +27,51 @@ prefix = "tikv"
...
 #cluster-id = 1
 # set pd address, host:port
-#pd = 
\ No newline at end of file
+#pd = 
+
+# For detailed explanation please refer to https://github.com/facebook/rocksdb/blob/master/include/rocksdb/options.h
+[rocksdb]
+# compression method (if any) is used to compress a block.
+#   kNoCompression
```
Comment (ngaut) [inline etc/config-template.toml]: I prefer to use No, Snappy, Zlib, Bzip2...

Comment (overvenus) [inline etc/config-template.toml]: The official options are kNoCompression, kSnappyCompression ... but it is ok for me to simplify those options if you insist.

Comment (ngaut) [inline etc/config-template.toml]: The original options are used in source code,  we are gonna use them as configuration. Let's keep things simple.

Comment (siddontang) [inline etc/config-template.toml]: +1
Btw, I prefer using lower case in configuration, like no, snappy ...

Comment (overvenus) [inline etc/config-template.toml]: Get it!

**Comment by ngaut** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**